File: hello-world/Z-Basic-Codes/C Prog/chapter 3/grade.c
#include<stdio.h>
int main(){
    char grade;
    int marks ;
    printf("enter marks");
    scanf("%d" ,&marks);
    if(marks<=100 && marks >=90)
    grade = 'A';
     else if (marks<=90 && marks>=80)
     grade = 'B';
     else if (marks<=80 && marks>=60)
     grade = 'C';
     else 
     grade = 'D';
     {printf(" the grade is:%c",grade);
     }
    return 0;
}

File: hello-world/PYTHON/Python/LECTURE 02/grade.py
marks=int(input("enter your marks  ::"))
if(marks>=90):
#print("A")
    grade="A"
elif(90>marks>=80):
    #print("B")
    grade="B"
elif(80>marks>=70):
    #print("C")
    grade="C"
else:
    grade="D"
    #print("D")
    print("your grade is:", grade)